Directory.Build.targets

```diff
@@ -18,19 +18,51 @@
     <Compile Include="$(AzureCoreSharedSources)AsyncLockWithValue.cs" LinkBase="Shared/Core" />
     <Compile Include="$(AzureCoreSharedSources)AzureKeyCredentialPolicy.cs" LinkBase="Shared/Core" />
     <Compile Include="$(AzureCoreSharedSources)ClientDiagnostics.cs" LinkBase="Shared/Core" />
+    <Compile Include="$(AzureCoreSharedSources)ChangeTrackingDictionary.cs" LinkBase="Shared/Core" />
```

Comment:
You will need to update [DownloadShareSource](https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/DownloadSharedSource.ps1#L35-L39) since we have a inclusion list of these files and this is checked at ci time to see that all of these files are in sync with the upstream.

Since you are using an alpha which won't be in main yet temporarily change the branch [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/DownloadSharedSource.ps1#L22) and [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/DownloadSharedSource.ps1#L23) so it can find all the new files in the new location.

Simply replace `git checkout` with `git checkout [branch]`